doc/shared/sunadaptcontroller/SUNAdaptController_ImExGus.rst

```diff
@@ -0,0 +1,129 @@
+..
+   Programmer(s): Daniel R. Reynolds @ SMU
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2023, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _SUNAdaptController.ImExGus:
+
+The SUNAdaptController_ImExGus Module
+======================================
+
+The ImEx Gustafsson implementation of the SUNAdaptController class, SUNAdaptController_ImExGus,
+implements a combination of two adaptivity controllers proposed
+by K. Gustafsson.  His "explicit" controller was proposed in :cite:p:`Gust:91`,
+is primarily useful with explicit Runge--Kutta methods, and has the form
+
+.. math::
+   h' \;=\; \begin{cases}
+      h_1\; \varepsilon_1^{-1/(p+1)}, &\quad\text{on the first step}, \\
+      h_n\; \varepsilon_n^{-k_1/(p+1)}\;
+        \left(\dfrac{\varepsilon_n}{\varepsilon_{n-1}}\right)^{k_2/(p+1)}, &
+      \quad\text{on subsequent steps}.
+   \end{cases}
+   :label: expGusController
+
+The default values of :math:`k_1` and :math:`k_2` are 0.367 and 0.268,
+respectively, and :math:`p` is the global order of the time
+integration method.  In this estimate, a floor of :math:`\varepsilon > 10^{-10}`
+is enforced to avoid division-by-zero errors.
```

Comment:
Maybe this and the next paragraph should be combined and this moved to the end since it applies to both controllers. 
```suggestion
integration method.  In this estimate, a floor of :math:`\varepsilon_* > 10^{-10}`
is enforced to avoid division-by-zero errors.
```